packages/node/src/indexer/blockDispatcher/worker-block-dispatcher.service.ts

```diff
@@ -194,8 +219,15 @@ export class WorkerBlockDispatcherService
         if (syncedDictionary) {
           this.syncWorkerRuntimes();
         }
+
+        if ((await worker.getMemoryLeft()) < 256) {
```

Comment:
This check shouldn't be necessary. `worker.waitForWorkerHeap(256)` should resolve instantly if memory is available